core-bundle/src/Resources/contao/classes/Messages.php

```diff
@@ -31,35 +31,6 @@ public function versionCheck()
 		return '';
 	}
 
-	/**
-	 * Check for maintenance mode
-	 *
-	 * @return string
-	 */
-	public function maintenanceCheck()
```

Comment:
We should probably keep the method, deprecate it and return an empty string.

Reply:
As I already said to @m-vo I don't see why anyone would've ever called this. It was a `getSystemMessages` hook listener so there's absolutely no valid use case to have called this method yourself.

Reply:
Ok, let's see if this assumption is correct. 😄